x-pack/plugins/uptime/public/components/queries/monitor_charts/monitor_charts.tsx

```diff
@@ -85,6 +85,10 @@ export const MonitorCharts = ({
           minDuration,
           status,
         }: any) => {
+          maxRtt.push({
```

Comment:
What does this do exactly? This could possibly use a comment.

Reply:
Addressed in https://github.com/elastic/kibana/pull/28217/commits/60c613c86103b694e789e19bb11941dce1f9a690.